### PR TITLE
VE 2064: Display count of non-portable infoboxes

### PR DESCRIPTION
**Pull request by pauloslund**

### CODE REVIEW

**Comment by inez** [comment]
I think that looks good, do you have any thoughts in regard to cache invalidation? Also, for how long it is being cached?

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php [hunk 1]
```diff
@@ -2,8 +2,9 @@
 class EmergencyBroadcastSystemController extends WikiaController {
 
 	public function index() {
-		if ( $this->isCorrectPage() && $this->isPowerUser() && $this->hasNonPortableInfoBoxes() && $this->canOpenEBS() ) {
-			$this->response->setVal( 'nonPortableCount', '3' ); // Temporary number for testing
+		$nonPortableCount = $this->countOfNonPortableInfoBoxes();
+		if ( $this->isCorrectPage() && $this->isPowerUser() && $this->canOpenEBS() && ( $nonPortableCount > 0 ) ) {
```
Comment (dianafa) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php]: What about moving `( $this->isCorrectPage() && $this->isPowerUser() && $this->canOpenEBS() )` check before we make the `getNonportableInfoboxes()` query?

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php [hunk 2]
```diff
@@ -2,8 +2,9 @@
...
+			$this->response->setVal( 'nonPortableCount', $nonPortableCount ); // Temporary number for testing
```
Comment (dianafa) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php]: is it still `// Temporary number for testing` ? ;)

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php (hunk 1 above)
Comment (inez) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php]: @dianafa Excellent point.

**Comment by pauloslund** [comment]
It's using CACHE_STANDARD, which is 24 hours. Haven't had a chance to look in to how they were invalidating the cache yet

**Comment by inez** [comment]
Please take a look at Insights/InsightsUnconvertedInfoboxesHooks.class.php I think that code should be modified due to your changes.

**Comment by pauloslund** [comment]
Good catch, I think I'm going to take another look at this PortableInfoboxQueryService

**Pull request merged**